db/models.py

```diff
@@ -14,3 +14,37 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(Actor)
+    genres = models.ManyToManyField(Genre)
```

Comment:
here is a recommendation about the naming of the related_name:
https://github.com/mate-academy/py-movie-moviesession-cinemahall/blob/master/checklist.md#code-style